bots/tools/meta_tools.py

```
@@ -67,40 +48,29 @@ def initialize_file_bot(file_name: str, system_message: str) -> str:
         os.makedirs(directory, exist_ok=True)
     name, _ = os.path.splitext(file_name)
-    file_bot = AnthropicBot(name=f"{name}")
+
+    if model =="haiku":
+        model_engine = Engines.CLAUDE45_HAIKU
+    elif model == "sonnet":
+        model_engine = Engines.CLAUDE46_SONNET
+    elif model == "opus":
+        model_engine = Engines.CLAUDE46_OPUS
+    else:
+        raise("Model string not found. Must be exactly haiku, sonnet, or opus")
```

Ruff (0.15.10)

[warning] Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)